cfgov/jinja2/v1/_includes/organisms/table.html

```diff
@@ -12,35 +12,44 @@
 
    ========================================================================== #}
 
+{% if value.data  %}
 <table class="o-table
               {{ 'o-table__row-links' if value.row_links else '' }}
-              table__stack-on-small
-              table__entry-header-on-small">
+              {{ 'table__stack-on-small' if value.is_stacked else '' }}
+              {{ 'table__striped' if value.is_striped else '' }}
+              {{ 'u-w100pct' if value.is_full_width else '' }}">
+    {% set row_index = 0 %}
+    {% if value.first_row_is_table_header and value.data | list | length > 0 %}
+    {% set row_index = 1 %}
     <thead>
+        {# Check to verify the row isn't empty #}
+        {% if value.data[0] | reject('none') | list | length %}
         <tr>
-            {% for thead in value.headers %}
-                <th>{{ thead }}</th>
+            {% for cell in value.data[0] %}
+            <th scope="col">
+                {{ cell if cell is not none else '' | trim | safe | linebreaksbr }}
+            </th>
             {% endfor %}
         </tr>
+        {% endif %}
     </thead>
-    {% for row in value.rows %}
+    {% endif %}
+    <tbody>
+    {% for row in value.data[row_index:] or [] %}
+        {% if row | reject('none') | list | length %}
         <tr>
-        {% for cell in row %}
-            {% set colcount = loop.index %}
-            {% for thead in value.headers %}
-                {% if colcount == loop.index %}
-                    {% if 'rich_text_blob' in cell.block_type %}
-                        <td data-label="{{ thead }}">
-                            {{ parse_links(cell.value) | safe }}
-                        </td>
-                    {% else %}
-                        <td data-label="{{ thead }}">
-                            {{ cell | safe }}
-                        </td>
-                    {% endif %}
+            {% for cell in row or [] %}
+                {% set tag = 'th scope="row"' if value.first_col_is_header and loop.first else 'td' %}
+                {% if value.data[0][loop.index0] is not none %}
+                   {% set data_label = 'data-label="{0}"'.format(value.data[0][loop.index0] | striptags | trim) %}
```

Comment:
Also, it looks like the data labels are displayed even if the table doesn't stack on mobile:

<img width="844" alt="screen shot 2016-09-22 at 3 07 19 pm" src="https://cloud.githubusercontent.com/assets/778171/18767835/5b5097c8-80d6-11e6-9ddb-e010d67e3472.png">

<img width="363" alt="screen shot 2016-09-22 at 3 07 29 pm" src="https://cloud.githubusercontent.com/assets/778171/18767842/6242905e-80d6-11e6-9573-a16503a1e430.png">

because of [this rule in enhancements/tables.less](https://github.com/cfpb/cfgov-refresh/blob/1db1f16329a030de487d7e3d61b038fb0a56b4a2/cfgov/unprocessed/css/enhancements/tables.less#L134-L142).